thirdeye/thirdeye-pinot/src/main/java/com/linkedin/thirdeye/common/ThirdEyeConfiguration.java

```diff
@@ -110,6 +110,10 @@ public String getAnomalyClassifierConfigPath() {
     return getRootDir() + "/detector-config/anomaly-functions/anomalyClassifier.properties";
   }
 
+  public String getCalendarApiKeyPath(){
+    return getRootDir() + "/holiday-loader-key.json";
+  }
```

Comment:
Remember the CSV data source? Make the path a (partial) URI and part of the configuration for easy testing. You can probably reuse some of the methods

Reply:
ok, we'll delay this a sdiscussed